profiler/src/ProfilerEngine/Datadog.Profiler.Native/CrashReporting.cpp

```diff
@@ -287,27 +306,31 @@ int32_t CrashReporting::SetMetadata(const char* libraryName, const char* library
     auto vecTags = ddog_Vec_Tag_new();
 
     const ddog_crasht_Metadata metadata = {
-        .library_name = libdatadog::to_char_slice(libraryName) ,
+        .library_name = libdatadog::to_char_slice(libraryName),
         .library_version = libdatadog::to_char_slice(libraryVersion),
         .family = libdatadog::to_char_slice(family),
-        .tags = &vecTags
-    };
+        .tags = &vecTags};
 
     for (int32_t i = 0; i < tagCount; i++)
     {
         auto tag = tags[i];
-        ddog_Vec_Tag_push(&vecTags, libdatadog::to_char_slice(tag.key), libdatadog::to_char_slice(tag.value));
+        auto result = ddog_Vec_Tag_push(&vecTags, libdatadog::to_char_slice(tag.key), libdatadog::to_char_slice(tag.value));
+        if (result.tag != DDOG_VEC_TAG_PUSH_RESULT_OK)
+        {
+            // Let's not stop right here.
+            ddog_Error_drop(&result.err);
```

Comment:
Honestly I think you can stop when adding a tag fails.

1/ that means something is terribly wrong
2/ we won't see the crash report if it's not properly tagged